### PR TITLE
Fix temp del

### PR DESCRIPTION
**Pull request by tralston**
Allows removing of files on Mac without hanging. Includes test.

### CODE REVIEW

File: src/temp.bash [hunk 1]
```diff
@@ -168,7 +168,7 @@ temp_del() {
 
   # Delete directory.
   local result
-  result="$(rm -r -- "$path" 2>&1)"
+  result="$(rm -rf -- "$path" 2>&1)"
```
Comment (sublimino) [inline src/temp.bash:171]: This instinctively makes me nervous : ] Should we add a check to ensure that `$path` doesn't equal `/` perhaps? Is there anything else we could do to protect from typos or accidental invocation?

**Comment by sublimino** [comment]
A few tests failing here please @tralston 🙏

**Comment by Vampire** [comment]
> A few tests failing here please @tralston 🙏

While it would be great if they get fixed,
the failing tests are not because of this PR, just if anyone wonders.
The build is failing since the last merge which was just a documentation change.
But the execution environment changed from trusty to xenial, so this could be actually a bug that happens on newer systems.

File: src/temp.bash (hunk 1 above)
Comment (martin-schulze-vireso) [reply src/temp.bash:175]: I tried what happens if you redirect `/dev/null` to `stdin` and it seems this is taken as implicit yes to the prompt. The prompt does not even show in this case and the return code is 0.

**Review by martin-schulze-vireso — APPROVED** [review]

**Comment by martin-schulze-vireso** [comment]
Thanks for your PR.

**Pull request merged**